### DISCUSSION TITLE
Alpha3 Version success!

### DISCUSSION THREAD

**Discussion by ferdiworks** [rrirower/DP2SceneGraphWizard#5] [General]
Today I downloaded and tested Alpha 3 version. I was able to run the software take the output zip file and sideload it with no errors. I was then able to package the App and upload to Roku where it past the Static Analysis and set deep linking and submit the channel for publishing!

Congrats Michael on an awesome product!

**Comment by rrirower** [comment]
Thanks, but, it's not done yet. There's more good stuff to come.

**Reply by ferdiworks** [reply]
Cant wait!
I published 3 channels today!

**Comment by ferdiworks** [comment]
I was trying to help one of the members in the dev forum, His feed of series and episodes would not parse in the software.
The feed is on amazon S3 and displays in a browser! The feed returns no errors!
But it doesn't parse!
https://s3.amazonaws.com/cdn1.jltv.tv/roku/jltv3_1.json

**Reply by rrirower** [reply]
I won't be able to take a look at this until Friday.

**Reply by ferdiworks** [reply]
ok sounds good

**Reply by rrirower** [reply]
I took another look. The feed file does not define categories. A category is defined by either a query or a playlist name. You can't simply define a playlist without associating it with a category.

**Reply by ferdiworks** [reply]
That was my thoughts exactly

**Comment by ferdiworks** [comment]
Question = I dont know why I never asked but maybe I overlooked something. How do we change the displayed text color of video descriptions or text displayed in general!

**Comment by rrirower** [comment]
There are two ways to do it.   Either in the .brs code or, in the associated .xml file that defines each screen.
